duecredit/stub.py

```diff
@@ -41,7 +41,7 @@ def nondecorating_decorator(func):
              return func
         return nondecorating_decorator
 
-    cite = load = add = _donothing
+    cite = load = add = citations_fromentrykey = _donothing
```

Comment:
that is why it shouldn't be there -- it is not really intended for user's consumption within the 'due'